docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -179,14 +179,69 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Composants" href="/fr/guides/components/" />
 </CardGrid>
 
+### Asides
+
+Les encarts (également connues sous le nom de "aside" ou encore "élément en marge") sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
```

Comment:
```suggestion
Les encarts (également connus sous le nom de « admonitions ») sont utiles pour afficher des informations secondaires à côté du contenu principal d'une page.
```

This feels a bit weird as we mix English and French here, and it looks like searching exactly for `"élément en marge"` does not bring any result related to asides. I think we should not stick to a 1:1 translation and just mention admonitions which [exists in the French dictionary](https://www.larousse.fr/dictionnaires/francais/admonition/1149).

Any thoughts

Reply:
I never seen "admonitions" in French, I usualy use _Asides_ and not _Encarts_, I searched how it's translated, except for inserts, I haven't really found better words.